cve_bin_tool/checkers/libxslt.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for libxslt
+https://www.cvedetails.com/product/14676/Xmlsoft-Libxslt.html?vendor_id=1962
+"""
+from . import Checker
+
+
+class LibxsltChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"libxslt.so.1.1.28.debug",
+        r"xsltNewSecurityPrefs : malloc failed",
+        r"xsltCheckWrite: out of memory for %s",
+        r"xsltCheckRead: URL parsing failed for %s",
+    ]
+    FILENAME_PATTERNS = [r"libxslt.so."]
+    VERSION_PATTERNS = [r"libxslt ([0-9]+\.[0-9]+\.[0-9]+)"]
```

Comment:
Quick reminder since you seem to be poking at this again: your signature is still broken.  You probably want

```
r"libxslt\.so\.([0-9]+\.[0-9]+\.[0-9]+)\.debug"
```

For at least the centos package.